src/component/Form.js

```diff
@@ -1,30 +1,35 @@
 import './styles/form.css';
-import { useDispatch, useSelector } from 'react-redux';
-import { useState } from 'react';
-import { addBook } from '../redux/book/booksSlice';
+import { useDispatch } from 'react-redux';
+import { useState, useEffect } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { addBooks, fetchBooks } from '../redux/book/booksSlice';
 
 export default function Form() {
   const [tit, setTitle] = useState('');
   const [auth, setAuthor] = useState('');
 
   const dispatch = useDispatch();
-  const { totalbooks } = useSelector((state) => state.book);
+  useEffect(() => {
+    dispatch(fetchBooks());
+  }, []);
   const book = () => {
-    const i = totalbooks.length + 1;
+    document.querySelector('.title').value = '';
+    document.querySelector('.author').value = '';
     const b = {
       title: tit,
       author: auth,
-      item_id: `item${i}`,
+      item_id: uuidv4(),
+      category: 'Economy',
     };
     return b;
   };
   return (
     <form>
-      <span>Add new book</span>
-      <div>
+      <span className="add">Add new book</span>
```

Comment:
- [ ] 👍 Good effort! To enhance the alignment of your button, kindly align it with the card component to the right side horizontally. By following this design adjustment, you'll improve the visual alignment and consistency of your project. Keep up the great work! 🌟👌

| Your | Design |
| --- | --- |
|  ![image](https://github.com/MayPyone/Bookstore/assets/55840999/19784afa-f3bd-4486-9358-5772559d9765)| ![image](https://github.com/MayPyone/Bookstore/assets/55840999/bb15f9f4-85b8-4ff2-b726-2be128c24e0a)|